sdks/python/apache_beam/io/gcp/bigquery_tools_test.py

```diff
@@ -50,6 +49,15 @@
 from apache_beam.io.gcp.internal.clients import bigquery
 from apache_beam.options.pipeline_options import PipelineOptions
 
+# Protect against environments where bigquery library is not available.
```

Comment:
(@pabloem is this something we want to do?)

Reply:
@apilloud this is copy from bigquery_test.py I assumed we want this consistent between files.

Reply:
Thanks for pointing that out. (I assumed we wouldn't want to do this in tests, but if we are doing it in others then follow that pattern.)